pagopa/pn-frontend
# Fix/pn 2204 remove explicit link in landing

## Short description
Removes explicit index.html in link in landing webapp footer

## How to test
- Start pn-landing-webapp
- click on 'Privacy Policy' link
- url should not end with index.html
